t/fcm-make/36-build-fail-cont-basic.t

```diff
@@ -43,9 +43,8 @@ sed -i 's/implicit none/implicit non/' src/greet_mod.f90  # introduce typo
 run_fail "$TEST_KEY" fcm make --new
 task_lines_from_log >"$TEST_KEY-log-tasks"
 file_cmp "$TEST_KEY-log-tasks" "$TEST_KEY-log-tasks" <<'__LOG__'
-[FAIL] compile   ???? ! greet_mod.o          <- greet_mod.f90
 [info] compile   ???? M world_mod.o          <- world_mod.f90
-[FAIL] compile   ---- ! hello.o              <- hello.f90
```

Comment:
I think the reverse sort order means that `hello.o` is not yet in the stack when it dies.